.github/workflows/translate-production.yml

```diff
@@ -0,0 +1,15 @@
+name: Translate Reports - Production
+
+on:
+  schedule:
+    - cron: "0 5 * * *" # Run every day at 5 AM GMT
+  
+  workflow_dispatch:
```

Comment:
This is to enable manual triggering or it is a leftover?

Reply:
Yes, it is to enable manual triggering if it's needed